sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulsePingSenderTests.java

```diff
@@ -92,11 +102,122 @@ void endpointChangesWithRedirectHeaderAndGetNewPingInterval() {
                 "instance1",
                 "machine1",
                 "qpid123",
-                "testSdkVersion");
+                "testSdkVersion",
+                quickPulseConfiguration);
         QuickPulseHeaderInfo quickPulseHeaderInfo = quickPulsePingSender.ping(null);
         assertThat(QuickPulseStatus.QP_IS_ON).isEqualTo(quickPulseHeaderInfo.getQuickPulseStatus());
         assertThat(1000).isEqualTo(quickPulseHeaderInfo.getQpsServicePollingInterval());
         assertThat("https://new.endpoint.com")
             .isEqualTo(quickPulseHeaderInfo.getQpsServiceEndpointRedirect());
+        assertThat(quickPulseConfiguration.getEtag()).isEqualTo("1::randometag::2::");
+    }
+
+
+    @Test
+    void successfulPingReturnsWithEtagHeader() {
```

Comment:
I am unclear how this test is different from the previous one.

Reply:
forgot to delete that test, should now be deleted.